src/Sections/Todos.php

```diff
@@ -13,14 +13,15 @@ class Todos extends AbstractSection
      * @param  string  $status
      * @return \Illuminate\Support\Collection
      */
-    public function index($page = null, $status = null)
+    public function index($page = null, $status = null, $completed = null)
     {
         $url = sprintf('buckets/%d/todolists/%d/todos.json', $this->bucket, $this->parent);
 
         $todos = $this->client->get($url, [
             'query' => [
                 'status' => $status,
                 'page' => $page,
+                'completed' => $completed ? 'true' : null,
```

Comment:
Thanks a lot for this PR, i didn't notice this parameter!

Can you do
```
'completed' => !!$completed
```
here instead?

If it works I'll merge it. Thx!

Reply:
Also: just to make sure that the options `null` and `false` returns the same results. If `false` returns every uncompleted tasks and `null` returns every tasks, you'd need to take that into consideration.

Reply:
Alright nevermind, I've tested this and the API only takes the boolean value as a string, and `null` or `false` returns the same thing, so your solution is the best.

I'm merging this, thank you!